src/components/Match/MatchStory.jsx

```diff
@@ -126,6 +126,15 @@ const formatList = (items, noneValue = []) => {
   }
 };
 
+// is given a hero in the format of npc_dota_hero_{heroname}, and returns the player corresponding to that hero, else undefined
+const heroToPlayer = (match, heroname) => match.players.find((player) => {
```

Comment:
I think the player_slot is populated on the server side using similar logic, so probably easier not to reimplement it